docs/docs/adding-markdown-pages.md

```diff
@@ -2,7 +2,162 @@
 title: Adding Markdown Pages
 ---
 
-This is a stub. Help our community expand it.
+Gatsby can easily convert Markdown pages to static pages in your site.
+It provides plugins to read and understand folders with markdown files and 
+node api, which help you create static pages automatically.
```

Comment:
My initial feedback is to make sure you write API with all caps. And are you referring to Node.js API? Because if so, then Node.js is capitalized too.